nixos/modules/services/misc/memos.nix

```diff
@@ -0,0 +1,175 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+let
+  cfg = config.services.memos;
+in
+{
+  options.services.memos = with lib; {
```

Comment:
Please drop unessessary `with lib` usage, as per https://github.com/NixOS/nixpkgs/issues/208242 and https://nix.dev/guides/best-practices#with-scopes.

It's fine to use something like `inherit (lib) mkOption types;` in the above let block, if the `lib` repetition is too much for you. Although IMO it's fine to just type out (e.g.) `lib.mkOption` each time 😉